### PR TITLE
feat(dev/Assignee) : create new task bug

### PR DESCRIPTION
**Pull request by Pendyala-thanvi**
### Functionality:
Create new task email input

### Solution:
Let's user give an email that is not present in datalist too.

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

File: src/app/services/tool/tools.service.ts [hunk 1]
```diff
@@ -26,12 +26,26 @@ export class ToolsService {
...
+      } else {
+        return undefined;
+      }
+    }
   }
 
+  validateEmail(email: string) {
+    const regularExpression = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return regularExpression.test(String(email).toLowerCase());
+   }
```
Comment (viv9k) [inline src/app/services/tool/tools.service.ts:47]: You can put this regx into validation service and from there we can validate email.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

File: src/app/services/tool/tools.service.ts [hunk 2]
```diff
@@ -26,10 +27,20 @@ export class ToolsService {
     return this.todayTime = hh + ":" + mn + ":" + ss;
   }
 
-  userName(name: string){
-    let startindex = name.indexOf('<')
-    let endindex = name.indexOf('>')
-    return name.substring(startindex+1,endindex);
+  getEmailString(email: any){
```
Comment (k0malSharma) [inline src/app/services/tool/tools.service.ts:30]: The assignee name is of type string

**Review by k0malSharma — CHANGES_REQUESTED** [review]
Please do the required changes

File: src/app/services/tool/tools.service.ts [hunk 3]
```diff
@@ -26,14 +26,21 @@ export class ToolsService {
     return this.todayTime = hh + ":" + mn + ":" + ss;
   }
 
-  userName(name: string){
-    let startindex = name.indexOf('<')
-    let endindex = name.indexOf('>')
-    return name.substring(startindex+1,endindex);
+  getEmailString(email: string){
+    let startindex = email.indexOf('<')
```
Comment (k0malSharma) [inline src/app/services/tool/tools.service.ts:30]: indexof() will not work on undefined values. So first check if email is undefined and return before performing indexOf().

**Review by k0malSharma — CHANGES_REQUESTED** [review]

**Review by k0malSharma — APPROVED** [review]
Ship it

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**